.github/workflows/branch.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       # NOTE - this doesn't currently work if the PR is coming from a fork, due to limitations in GitHub actions secrets
       - name: Post PR comment
         if: failure()
-        uses: mshick/add-pr-comment@v2
+        uses: mshick/add-pr-comment@b8f338c590a895d50bcbfa6c5859251edc8952fc # v2
```

Comment:
Curious why this was needed?

Reply:
Thanks. It's part of the latest nf-core updates (e.g., https://github.com/nf-core/tools/pull/2739)